doc/themes/the-things-stack/layouts/shortcodes/frequency_plans.html

```diff
@@ -0,0 +1,15 @@
+{{ $package := or (.Get "package") "frequency-plans" }}{{ $frequency_plan := .Get "frequency-plan" -}}
```

Comment:
This file could use a little rewrite. I can do that when I have time.

Reply:
:) thanks

Reply:
i also snuck some badness in to the glossary.html shortcode, it could use the same cleanup #2563